src/datatable/DatatableODSConverter.js

```diff
@@ -0,0 +1,101 @@
+import { DefaultDOMElement } from 'substance'
+import JSZip from 'jszip'
+
+import DatatableConverter from './DatatableConverter'
+
+/**
+ * Converter to import/export a Datatable from/to an Open Document Spreadsheet (ODS) file
+ *
+ */
+export default class DatatableODSConverter extends DatatableConverter {
+
+  /**
+   * @override
+   */
+  match (path) {
+    let {ext} = this._parsePath(path)
+    return Promise.resolve(ext === 'ods')
+  }
+
+  /**
+   * @override
+   */
+  import (path, storer, buffer) {
+    return storer.readFile(path).then(data => {
+      return JSZip.loadAsync(data)
+    }).then(zip => {
+      return zip.file('content.xml').async('string')
+    }).then(content => {
+      let $datatable = this._importDatatableFromContent(content)
+      return this._importWriteBuffer($datatable, buffer)
+    })
+  }
+
+  /**
+   * Helper method to convert an ODS `<office:document-content>` root element
+   * into a Datatable `<datatable>` element.
+   * 
+   * @param  {string} content - XML string of the `content.xml` file
+   * @return {DOMElement}
+   */
+  _importDatatableFromContent(content) {
+    let {$datatable, $$} = this._importCreateElement()
+    let $fields = $$('fields')
+    let $values = $$('values')
+
+    // This suceeds in the Node.js but in the browser fails to parse
```

Comment:
@oliver---- : there seems to be an issue parsing XML with namespaced tagName (?)